packages/web/src/scss/components/Tag/README.md

```diff
@@ -3,50 +3,32 @@
 Displays an inline element as a tag. Tags can be used to highlight or add emotion
 to information. The base tag component styles the text, adds padding, corners and
 background. Tags come in a variety of colors that can be set using modifier classes.
-All colors have light and dark variants.
+All colors have light and dark themes. Tags have three sizes: xsmall, small and medium.
```

Comment:
This is inconsistency in design system. When there are three variants of size, I consider that the values should be `small`, `medium`, `large`, for better usage. But this is weird. Just for notice.

Reply:
I will discuss it with the designers, but I assume we won't change it now. But they may consider it in the future.

Reply:
Tags themselves are rather bigger than may be expected, however their names correspond to the font sizes used.